action-run.sh

```diff
@@ -14,6 +14,12 @@ if [ "${CI}" = "true" ]; then
   set -e
 fi
 
+## Freeing space since 14GB are not enough anymore
+df -h
+echo "Deleting android, dotnet, haskell, CodeQL, Python, swift to free up space"
+sudo rm -rf /usr/local/lib/android /usr/share/dotnet /usr/local/.ghcup /opt/hostedtoolcache/CodeQL /opt/hostedtoolcache/Python /usr/share/swift
+df -h
+
```

Comment:
Can we add it to the `if [ "${CI}" = "true" ]; then`?

In case we need at some point to run this locally to not start to remove without permission.

Reply:
Good call, I should have though about that corner case